<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchViewWidget</class>
 <widget class="QWidget" name="BatchViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>692</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Batch View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="printScanButton">
         <property name="text">
          <string>Print Scan</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/icons/printscan.png</normaloff>:/icons/icons/printscan.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printBatchButton">
         <property name="text">
          <string>Print Batch</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/icons/printbatch.png</normaloff>:/icons/icons/printbatch.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="delayLabel">
           <property name="text">
            <string>Delay</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="delayDoubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="specialValueText">
            <string>Off</string>
           </property>
           <property name="suffix">
            <string> μs</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="highPassFilterLabel">
           <property name="text">
            <string>High Pass</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="highPassFilterSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="specialValueText">
            <string>Off</string>
           </property>
           <property name="suffix">
            <string> kHz</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="exponentialFilterLabel">
           <property name="text">
            <string>Exp. Filter</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="exponentialFilterDoubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="specialValueText">
            <string>Off</string>
           </property>
           <property name="suffix">
            <string> μs</string>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="removeDCCheckBox"/>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="removeDCLabel">
           <property name="text">
            <string>Remove DC</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="zeroPadFIDsCheckBox"/>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="zeroPadFIDsLabel">
           <property name="text">
            <string>Zero-pad FIDs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="reprocessButton">
         <property name="text">
          <string>Reprocess</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="batchSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="AnalysisWidget" name="analysisWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
        </widget>
        <widget class="PeakListWidget" name="peakListWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPrint_Scan">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/printscan.png</normaloff>:/icons/icons/printscan.png</iconset>
   </property>
   <property name="text">
    <string>Print Scan</string>
   </property>
   <property name="toolTip">
    <string>Print the currently displayed scan</string>
   </property>
  </action>
  <action name="actionPrint_Batch">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/printbatch.png</normaloff>:/icons/icons/printbatch.png</iconset>
   </property>
   <property name="text">
    <string>Print Batch</string>
   </property>
   <property name="toolTip">
    <string>Print the displayed batch scan</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalysisWidget</class>
   <extends>QWidget</extends>
   <header>analysiswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PeakListWidget</class>
   <extends>QWidget</extends>
   <header>peaklistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
